artemis-hawtio/artemis-console/pom.xml

```diff
@@ -76,9 +76,14 @@
         <artifactId>maven-war-plugin</artifactId>
         <configuration>
           <useCache>true</useCache>
+          <!-- exclude these jars as they are already shipped with the broker -->
           <packagingExcludes>
             **/lib/slf4j*.jar,
-            **/lib/log4j*.jar
+            **/lib/log4j*.jar,
+            **/lib/guava*.jar,
+            **/lib/commons-io*.jar,
+            **/lib/commons-logging*.jar,
+            **/lib/failureaccess*.jar
```

Comment:
The comment added above this is "\<!-- exclude these jars as they are already shipped with the broker --\>"

I think we need to be very clear what is meant by 'shipped with the broker' here, and in turn what it is is being expected/required that a user might need to provide to make the resulting war usable in their situation.

The broker `artemis-server` module does not depend on or provide guava or commons-io, and neither does the `artemis-web` module this war is typically used via. The assembly distribution does include those things, but only due to _other_ included bits bringing them in, not the broker. So if we completely remove those deps, they wont in fact be provided necessarily, i.e this actually is likely to actually break non-assembly use cases.

Previously we have entirely stripped some logging deps, but the _required_ API dep was certain to provided, either by the modules the war is most likely to be used with, or by the application itself (as is a logging implementation unless they have none configured). That isnt necessarily true for the other deps like these.

Reply:
I changed up the strategy. Instead of dealing with this when building `console.war` I am dealing with it when creating the broker distribution. That should eliminate any problems with non-assembly use-cases.